dashboard/config/initializers/devise.rb

```diff
@@ -190,7 +190,7 @@
   # ==> Configuration for :timeoutable
   # The time you want to timeout the user session without activity. After this
   # time the user will be asked for credentials again. Default is 30 minutes.
-  # config.timeout_in = 30.minutes
+  config.timeout_in = 60.days
```

Comment:
Curious how you arrived at 60 days for the default? I know we [configure our session store to expire after 40.days](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/initializers/session_store.rb#L12). I would imagine that expiration would get hit before this does, so it probably doesn't matter, I was just curious.

Reply:
I'd definitely like for us to implement this in a way that enforces consistency between our devise configuration and our session store configuration. And, perhaps more importantly, in a way which will make it clear to anyone who wants to do work on one of them in the future that they should also make sure to consider the other one